php-frontend/src/main/java/org/sonar/php/api/PHPKeyword.java

```diff
@@ -25,6 +25,7 @@
 
 public enum PHPKeyword implements TokenType, GrammarRuleKey {
 
+  // todo (Lena) : should be removed as HALT_COMPILER_STATEMENT is represented by function call statement
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)